PokemonGo.RocketAPI/POGOProtos/Networking/Envelopes/Signature.cs

```diff
@@ -0,0 +1,2580 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: POGOProtos/Networking/Envelopes/Signature.proto
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
+using pb = global::Google.Protobuf;
+using pbc = global::Google.Protobuf.Collections;
+using pbr = global::Google.Protobuf.Reflection;
+using scg = global::System.Collections.Generic;
+namespace POGOProtos.Networking.Signature {
+
+  /// <summary>Holder for reflection information generated from POGOProtos/Networking/Envelopes/Signature.proto</summary>
+  public static partial class SignatureReflection {
+
+    #region Descriptor
+    /// <summary>File descriptor for POGOProtos/Networking/Envelopes/Signature.proto</summary>
+    public static pbr::FileDescriptor Descriptor {
+      get { return descriptor; }
+    }
+    private static pbr::FileDescriptor descriptor;
+
+    static SignatureReflection() {
+      byte[] descriptorData = global::System.Convert.FromBase64String(
+          string.Concat(
+            "Ci9QT0dPUHJvdG9zL05ldHdvcmtpbmcvRW52ZWxvcGVzL1NpZ25hdHVyZS5w",
```

Comment:
This whole file is compiled, I didn't paste anything in there. If you're curious, this is a protofile descriptor which is needed when you are working with message types not known at compile-time or you want to access fields at runtime using reflection. Either way, this is not needed in this application since we are already using the compiled classes, it's only there because the protobuf compiler did.